src/pages/Details/components/SubscribeOptions.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import './SubscribeOptions.scss';
+
+const selectList = [
```

Comment:
```suggestion
const SELECT_LIST = [
```

상수데이터 이름은 UPPER_CASE 로 적용해주세요!